test/general/signature.js

```diff
@@ -1493,28 +1409,6 @@ hkJiXopCSWKSlQInL1devkJJUWJmTmZeugJYlpdLAagQJM0JpsCqIQZwKgAA
     });
   });
 
-  it('Verify test with expired verification public key', async function() {
```

Comment:
I didn't get what these 2 tests were checking, given in neither case verification is expected to fail...?
I think the tests I added should cover all "expired key" cases anyway.